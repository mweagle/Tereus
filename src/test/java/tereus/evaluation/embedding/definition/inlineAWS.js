CloudFormationTemplate("Test")({
    "Description": "Test",
    "Resources": {
        "Templated": Embed.File("resources/inlineAWS.txt")
    },
    "Outputs": {}
});
