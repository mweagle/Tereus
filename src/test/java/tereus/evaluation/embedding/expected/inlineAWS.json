{
  "Description": "Test",
  "Resources": {
    "Templated": {
      "Fn::Join": [
        "",
        [
          "Include inline ",
          {
            "Ref": "AWS::Region"
          },
          " pseudo params for your ",
          {
            "Ref": "AWS::AccountId"
          },
          " template\n"
        ]
      ]
    }
  },
  "Outputs": {},
  "AWSTemplateFormatVersion": "2010-09-09"
}